ios/Framework/Core/Asset/DisplayScreenlet/AssetDisplayScreenlet.swift

```diff
@@ -122,10 +122,16 @@ import UIKit
 
 	public func configureInnerScreenlet(innerScreenlet: BaseScreenlet, asset: Asset) {
 		if let screenlet = innerScreenlet as? FileDisplayScreenlet {
-			screenlet.fileEntry = FileEntry(attributes: asset.attributes)
 			screenlet.autoLoad = false
 			screenlet.offlinePolicy = self.offlinePolicy
 			screenlet.presentingViewController = self.presentingViewController
+
+			if let image = asset as? ImageEntry {
+				screenlet.classPK = image.imageEntryId
```

Comment:
The className is set by default in FileDisplayScreenlet so you don't need to set it here.